documentation/release_notes.markdown

```diff
@@ -14,6 +14,104 @@ canonical: "/puppetdb/latest/release_notes.html"
 [metrics]: ./api/metrics/v1/changes-from-puppetdb-v3.html
 [pqltutorial]: ./api/query/tutorial-pql.html
 
+4.0.2
+-----
+
+PuppetDB 4.0.2 is a backward-compatible bugfix release that should
+improve report processing performance, and fix a node-expiration bug,
+a bug in the handling of some correlated subqueries, and a bug that
+disallowed colons in request routes.
+
+### Bug Fixes and Maintenance
+
+* Report processing should be more efficient, after the removal of
+  some redundant validations.
+  ([PDB-2620](https://tickets.puppetlabs.com/browse/PDB-2620))
+
+* Colons in routes should be allowed again (e.g. resources
+  types/titles, etc.).  This was broken by the 4.0.0 release.
+  ([PDB-2624](https://tickets.puppetlabs.com/browse/PDB-2624))
+
+* Correlated subqueries (PQL or AST) involving fact values should work
+  better.  Previously a query like this this would fail
+  ([PDB-2474](https://tickets.puppetlabs.com/browse/PDB-2474)):
+
+```
+    resource {
+      type = 'Package'
+      and title in facts [value] { name = 'apache_package_name' }
+    }
+```
+
+* Nodes should no longer be incorrectly eligible for expiration in an
+  unlikely corner case.
+  ([PDB-2617](https://tickets.puppetlabs.com/browse/PDB-2617))
+
+* PuppetDB has migrated to ezbake 0.3.24.
+  ([PDB-2645](https://tickets.puppetlabs.com/browse/PDB-2645))
+
+* Support for Debian 6 is no longer being tested.
```

Comment:
Maybe we should instead say Debian 6 has been EOL'd and as such is no longer being tested?